From 52833d992727a404a539d4b823c0ee8c24b6ae00 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:26:39 -0400
Subject: [tools] perf: Fix non-debug build

Message-id: <1408462094-14194-231-git-send-email-jolsa@redhat.com>
Patchwork-id: 88203
O-Subject: [PATCH RHEL7.1 BZ1131394 230/325] perf tools: Fix non-debug build
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 74af377bc25dd9ebcb0be12836abb6b401b5dd08
Author: Adrian Hunter <adrian.hunter@intel.com>
Date: Tue Oct 22 10:34:05 2013 +0300

description
===========
In the absence of s DEBUG variable definition on the command line perf
tools was building without optimization.  Fix by assigning DEBUG if it
is not defined.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/config/Makefile b/tools/perf/config/Makefile
index c516d6b..543aa95 100644
--- a/tools/perf/config/Makefile
+++ b/tools/perf/config/Makefile
@@ -66,6 +66,10 @@ ifneq ($(WERROR),0)
   CFLAGS += -Werror
 endif
 
+ifndef DEBUG
+  DEBUG := 0
+endif
+
 ifeq ($(DEBUG),0)
   CFLAGS += -O6
 endif
-- 
1.7.1